web_src/js/components/RepoActionView.vue

```diff
@@ -676,22 +671,6 @@ export function initRepositoryActionView() {
 
 /* selectors here are intentionally exact to only match fullscreen */
 
-.full.height > .action-view-right {
```

Comment:
I also ever questioned about this 😂

* https://github.com/go-gitea/gitea/pull/24876#discussion_r1209161891


Glad to see they are moved to a global scope.

Reply:
Yes I was surprised it even worked, but apparently "scoped" style only means the final part of the selector needs to be local to the component, not the whole selector. Pretty bad decision from Vue authors imho.